kfs-core/src/main/java/edu/arizona/kfs/gl/service/impl/ScrubberValidatorImpl.java

```diff
@@ -219,16 +220,15 @@ protected Message validateGTEAccount(OriginEntryFull originEntry) {
         account.refreshReferenceObject("subFundGroup");
```

Comment:
This is specifically how it was ported form v3, so my assumption is yes, that the explicit reference refreshes were needed further down to detect between a real absence and just lazy loading.